src/netius/extra/smtp_a.py

```
@@ -0,0 +1,192 @@
+            status,
+            froms,
+            self.activity_url,
+        )
+        try:
+            netius.clients.HTTPClient.method_s(
+                "POST",
+                self.activity_url,
+                headers=headers,
+                data=data,
+            )
+        except Exception as exception:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)